cf_remote/packages.py

```diff
@@ -155,7 +156,7 @@ def init_download(self):
                 self.artifacts.append(artifact)
 
     def find(self, tags, extension=None):
-        if not self.extended_data:
+        if self.extended_data is None:
             self.init_download()
```

Comment:
After your last force push here, tests are failing, so more work needed.

Reply:
nice to have tests :)